packages/actor-sparql-serialize-stats/lib/ActorSparqlSerializeStats.ts

```diff
@@ -0,0 +1,73 @@
+import {IActorQueryOperationOutputBindings,
+  IActorQueryOperationOutputQuads} from "@comunica/bus-query-operation";
+import {ActorSparqlSerializeFixedMediaTypes, IActionSparqlSerialize,
+  IActorSparqlSerializeFixedMediaTypesArgs, IActorSparqlSerializeOutput} from "@comunica/bus-sparql-serialize";
+import {Readable} from "stream";
+
+/**
+ * Serializes SPARQL results for testing and debugging.
+ */
+export class ActorSparqlSerializeStats extends ActorSparqlSerializeFixedMediaTypes {
+
+  private startTime: [number, number] = process.hrtime();
```

Comment:
Not sure what kind of behaviour we want here.

Now, the start time will be the point in time at which the comunica engine is initialized. Every query that is executed in this engine will use this same start time.

Perhaps what we want is the start time at which a query is initialized? In this case, this start time would have to be initialized in the `runHandle ` method.